test/testground/network/sync.go

```diff
@@ -132,15 +132,13 @@ func DownloadNodeConfigs(ctx context.Context, runenv *runtime.RunEnv, initCtx *r
 // GenesisWrapper is a simple struct wrapper that makes it easier testground to
```

Comment:
I think the comment removals in this file are accidental and should prob be reverted.